bin/easy_install

```diff
@@ -0,0 +1,8 @@
+#!/Users/nmint8m/Repositories/Nimble/flutter_templates/bin/python3
```

Comment:
we should put `./bin` into `.gitignore`

Reply:
@nmint8m sorry for not clear enough 😓 we also need to remove `bin/` folder out of the version control 🙏